integration_tests/main_test.go

```diff
@@ -27,6 +27,7 @@ func TestMain(m *testing.M) {
 	opts := []goleak.Option{
 		goleak.IgnoreTopFunction("github.com/pingcap/goleveldb/leveldb.(*DB).mpoolDrain"),
 		goleak.IgnoreTopFunction("github.com/tikv/client-go/v2/txnkv/transaction.keepAlive"), // TODO: fix ttlManager goroutine leak
+		goleak.IgnoreTopFunction("github.com/tikv/client-go/v2/config/retry.(*Config).createBackoffFn.newBackoffFn.func2"),
```

Comment:
Same with upper line, cause by `ttlManager`.

```
2024-04-22T09:15:14.9274088Z PASS
2024-04-22T09:15:15.3698214Z goleak: Errors on successful test run: found unexpected goroutines:
2024-04-22T09:15:15.3700768Z [Goroutine 8419 in state select, with github.com/tikv/client-go/v2/config/retry.(*Config).createBackoffFn.newBackoffFn.func2 on top of the stack:
2024-04-22T09:15:15.3703402Z github.com/tikv/client-go/v2/config/retry.(*Config).createBackoffFn.newBackoffFn.func2({0x242a350, 0x40874a0}, 0xffffffffffffffff)
2024-04-22T09:15:15.3704673Z 	/home/runner/work/client-go/client-go/config/retry/config.go:205 +0x505
2024-04-22T09:15:15.3706063Z github.com/tikv/client-go/v2/config/retry.(*Backoffer).BackoffWithCfgAndMaxSleep(0xc000dda750, 0xc000476600, 0x1e70420?, {0x2410740, 0xc0033b21c8})
2024-04-22T09:15:15.3707218Z 	/home/runner/work/client-go/client-go/config/retry/backoff.go:195 +0x66d
2024-04-22T09:15:15.3708225Z github.com/tikv/client-go/v2/config/retry.(*Backoffer).Backoff(0xc000dda750, 0xc000476600, {0x2410740, 0xc0033b21c8})
2024-04-22T09:15:15.3709208Z 	/home/runner/work/client-go/client-go/config/retry/backoff.go:122 +0x251
2024-04-22T09:15:15.3710510Z github.com/tikv/client-go/v2/tikv.(*KVStore).getTimestampWithRetry(0xc001eed1e0, 0xc000dda750, {0x20b5997, 0x6})
2024-04-22T09:15:15.3711460Z 	/home/runner/work/client-go/client-go/tikv/kv.go:465 +0x42b
2024-04-22T09:15:15.3712413Z github.com/tikv/client-go/v2/tikv.(*KVStore).GetTimestampWithRetry(0xc000ad8ab8?, 0xc000ad8a28?, {0x20b5997?, 0x0?})
2024-04-22T09:15:15.3713314Z 	/home/runner/work/client-go/client-go/tikv/kv.go:440 +0x18
2024-04-22T09:15:15.3714625Z github.com/tikv/client-go/v2/txnkv/transaction.keepAlive(0xc00206c800, 0xc00137e540, {0xc003414678, 0x4, 0x4}, 0x0, 0x1)
2024-04-22T09:15:15.3715835Z 	/home/runner/work/client-go/client-go/txnkv/transaction/2pc.go:1204 +0x2b7
2024-04-22T09:15:15.3716668Z created by github.com/tikv/client-go/v2/txnkv/transaction.(*ttlManager).run in goroutine 8369
2024-04-22T09:15:15.3717487Z 	/home/runner/work/client-go/client-go/txnkv/transaction/2pc.go:1161 +0x1d1
```

Reply:
I removed this ignore and no leak happens in the test, it might by fix by [this change](https://github.com/tikv/client-go/commit/edba2544d5dcbe0cf6f8679bddabd321752debdc#diff-b40fa902c918781256fba826f2b15c731340e6068e77ebb11c494d9ba54a446cR190).